### PR TITLE
(SIMP-4009) Add support for OEL and Puppet 5

### PR DESCRIPTION
**Pull request by trevor-vaughan**
SIMP-4009 #comment updated compliance_markup

### CODE REVIEW

File: data/compliance_profiles/OracleLinux/6/nist_800_53_rev4.json [hunk 1]
```diff
@@ -0,0 +1,3024 @@
+{
```
Comment (heliocentric) [inline data/compliance_profiles/OracleLinux/6/nist_800_53_rev4.json]: These files should be symlinks if they are identical

Comment (trevor-vaughan) [reply data/compliance_profiles/OracleLinux/6/nist_800_53_rev4.json]: Symlinks get rejected by `puppet module build` and the Forge

Comment (heliocentric) [reply data/compliance_profiles/OracleLinux/6/nist_800_53_rev4.json]: lovely! quality work. have to push up v2 soon and write a splitting script so the duplication and syncing problems can stop.

**Comment by jeefberkey** [comment]
https://gitlab.com/simp/pupmod-simp-compliance_markup/pipelines/26101291

**Comment by jeefberkey** [comment]
https://gitlab.com/simp/pupmod-simp-compliance_markup/pipelines/26824734

**Comment by jeefberkey** [comment]
https://gitlab.com/simp/pupmod-simp-compliance_markup/pipelines/26824979

**Comment by jeefberkey** [comment]
https://gitlab.com/simp/pupmod-simp-compliance_markup/pipelines/26825082

**Comment by ayohrling** [comment]
Tests are failing on OEL for acceptance. Checking out reasoning now. It looks like it's an actual issue with functionality.

**Comment by ayohrling** [comment]
Alright, so I seem to have isolated the issue at least somewhat. It is spec related. It's not creating the /etc/puppetlabs/puppet/hiera.yaml file with the expected values in this stanza: 

```
it 'should work with no errors' do
            create_remote_file(host, '/etc/puppetlabs/puppet/hiera.yaml', hiera_yaml)
            create_remote_file(host, '/etc/puppetlabs/code/hieradata/default.yaml', base_hieradata.to_yaml)
            apply_manifest_on(host, base_manifest, :catch_failures => true)
          end
```

**Comment by ayohrling** [comment]
https://gitlab.com/simp/pupmod-simp-compliance_markup/pipelines/27136875

**Pull request merged**